src/test/scala/mirrorAndCheck.scala

```diff
@@ -0,0 +1,11 @@
+package ohnosequences.db.ncbitaxonomy.test
+
+import org.scalatest.{CancelAfterFailure, Suites}
+
+class MirrorAndCheck
+    extends Suites(
+      // Ensure mirroring before checking files
+      new Mirror,
+      new CheckOldVersions,
+      new CheckFullTaxonomy
+    )
```

Comment:
Ah, ok, this and the `DoNotDiscover` annotations make sure the order is right, have I understood it right?

Reply:
The `Suites` structure makes it possible to order the tests, provided that parallel execution of tests is not run (we have it disabled in our default configuration, so yay :tada: ). The `DoNotDiscover` tells `scalatest` to not execute a suite that is marked with that annotation (if we did not include the `DoNotDiscover`, `Mirror` would be executed one time inside the `Suites` structure and another time as a normal suite, as the rests of tests do).

Reply:
Ok, thanks!